cfme/cloud/provider/__init__.py

```diff
@@ -75,6 +75,23 @@ def is_displayed(self):
     including_entities = View.include(VMEntities, use_parent=True)
 
 
+class CloudProviderTenantsView(BaseLoggedInPage):
+    """
+    The collection page for cloud tenants
+    """
+    @property
+    def is_displayed(self):
+        return (
+            self.breadcrumb.locations[0] == 'Cloud Providers' and
```

Comment:
Optional:
I would go with `self.breadcrumb.active_location`. It will be unique check. If we go with `locations[0]` there may be a chance that other view can satisfy this condition means `Datastore` or `Other`...